### PR TITLE
Pin documentation version to 1.0.0

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,8 +81,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <!-- Current plugin version -->
-        <fabric8.vertx.plugin.version>${project.version}</fabric8.vertx.plugin.version>
+        <!-- Current stable plugin version -->
+        <fabric8.vertx.plugin.version>1.0.0</fabric8.vertx.plugin.version>
```
Comment (kameshsampath) [inline pom.xml]: @rhuss dont use the property `fabric8.vertx.plugin.version` this property is used as part of the $project.basedir/src/main/resources/vertx-maven-plugin.properties to identify the current plugin version which is then used in various test cases. thats why I see the tests failing.  I will update this property to another property name and push it back

**Comment by kameshsampath** [comment]
closing this PR with new update that will prevent the tests from failing

**Comment by kameshsampath** [comment]
i guess we need to update the circle.yml to generate website only from tag, i don't think so we need to have another property. 

thoughts?

**Comment by rhuss** [comment]
I close this one in favor of #57

**Pull request closed without merging**